cookbook/bundles/extension.rst

```diff
@@ -15,7 +15,7 @@ Creating an Extension Class
 ---------------------------
 
 In order to load service configuration, you have to create a Dependency
-Injection Extension for your bundle. This class has some conventions in order
+Injection (DI) Extension for your bundle. This class has some conventions in order
```

Comment:
We could also just avoid using DI in the docs (which imho were a better practice). But as long as we use them it's imho indeed a good idea to introduce it properly.